app/src/components/misc/Favicon/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { Avatar } from "antd";
 import React, { CSSProperties, useEffect, useMemo, useState } from "react";
-import { addUrlSchemeIfMissing } from "views/features/api-client/apiUtils";
 import { debounce } from "lodash";
+import { addUrlSchemeIfMissing } from "features/apiClient/screens/apiClient/apiUtils";
```

Comment:
This is being used outside the APIClient's feature folder. Can we export it from the feature folder's root index.ts file?

Reply:
Ideally this should not be here, for now leaving this as it is, adding a todo.